src/components/DpDataTable/DpDataTable.vue

```diff
@@ -715,6 +715,8 @@ export default {
     }
 
     this.forceElementSelections(this.shouldBeSelectedItems)
+
+    this.headerCellCount = this.headerFields.length
```

Comment:
since this is legit, I wonder why headerCellCount is no computed...

Reply:
With DpDataTable, I'm always afraid of unintentionally breaking things, so I don't feel very adventurous here.. :see_no_evil:

Reply:
true